src/test/java/subway/stations/StationRestAssured.java

```diff
@@ -0,0 +1,22 @@
+package subway.stations;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.http.MediaType;
+
+import java.util.Map;
+
+public class StationRestAssured {
+
+    private static final String apiPath = "/stations";
```

Comment:
```suggestion
    private static final String API_PATH = "/stations";
```
일반적으로 상수는 대문자에 단어사이는 언더스코어(_)로 구분하는 형태로 많이 사용합니다 😄 

[Google Style Guide](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)